### PR TITLE
Forget Password #4

### PR DESCRIPTION
**Pull request by Sahani13**
Implemented forget password code.

### CODE REVIEW

File: Frontend/src/screens/FogotPassword.js [hunk 1]
```diff
@@ -1,15 +1,53 @@
...
-      <Text>Fogot password screen</Text>
-      </View>
-    </BackgroundFlex>
 
+      
+      <HeaderTextComponent title="Reset Password" />
+
+      
+      <View style={{margin:30}} ></View>
+      <ImageComponentAuth source={require('../../assets/Reset_Pasword.png')}/>
```
Comment (Pradeepkumara95) [inline Frontend/src/screens/FogotPassword.js:25]: Image is missing, please commit the right image with the same name written here.

File: Frontend/src/screens/FogotPassword.js [hunk 2]
```diff
@@ -1,15 +1,53 @@
...
+import HeaderTextComponent from '../components/HeaderTextComponent';
 import BackgroundFlex from '../components/BackgroundFlex';
 
-export default function FogotPasswordSC({ navigation }) {
+
+export default function FogotPasswordScreen({ navigation }) {
+  const [email, setEmail] = useState('');
+  
+  const handleFogetPasword = () => {
+    navigation.navigate('Dashboard');
```
Comment (Pradeepkumara95) [inline Frontend/src/screens/FogotPassword.js:14]: redirect to the login page

File: Frontend/src/screens/FogotPassword.js [hunk 3]
```diff
@@ -1,15 +1,53 @@
-import React from 'react';
-import { View, Text } from 'react-native';
+import React, { useState } from 'react';
+import { Text, TouchableOpacity, View, StyleSheet } from 'react-native';
```
Comment (Pradeepkumara95) [inline Frontend/src/screens/FogotPassword.js:2]: import {Alert}

File: Frontend/src/screens/FogotPassword.js [hunk 4]
```diff
@@ -1,15 +1,53 @@
...
+import ImageComponentAuth from '../components/ImageComponentAuth';
+import HeaderTextComponent from '../components/HeaderTextComponent';
 import BackgroundFlex from '../components/BackgroundFlex';
 
-export default function FogotPasswordSC({ navigation }) {
+
+export default function FogotPasswordScreen({ navigation }) {
+  const [email, setEmail] = useState('');
+  
+  const handleFogetPasword = () => {
```
Comment (Pradeepkumara95) [inline Frontend/src/screens/FogotPassword.js:13]: Alert.alert(
      "Title",
      "Please check your e mail",)

**Review by Pradeepkumara95 — CHANGES_REQUESTED** [review]
Add the popup alert to the button component. follow comments

**Review by sarasikarunanayaka — APPROVED** [review]
Forget password page UI implementation is running without any errors.

**Review by Pradeepkumara95 — APPROVED** [review]
UI is completed

**Pull request merged**